### PR TITLE
fix high severity vulnerabilities and version bumps

### PR DESCRIPTION
**Pull request by johnduffell**
This PR fixes up several high severity vulnerabilities, plus a couple of incidental version bumps.

Tested locally and the server starts and serves a basic request

Tested in CODE and it seems to handle the main attributes endpoint fine.

### CODE REVIEW

File: membership-attribute-service/app/monitoring/SentryLogging.scala [hunk 1]
```diff
@@ -19,10 +18,12 @@ object SentryLogging extends SanitizedLogging {
...
         logger.info(s"Initialising Sentry logging")
         Try {
-          val sentryClient = Sentry.init(sentryDSN)
+          Sentry.init(sentryDSN)
           val buildInfo: Map[String, String] = app.BuildInfo.toMap.view.mapValues(_.toString).toMap
           val tags = Map("stage" -> config.stage) ++ buildInfo
-          sentryClient.setTags(tags.asJava)
+          tags.foreach { case (key, value) =>
+            Sentry.setTag(key, value)
+          }
```
Comment (johnduffell) [inline membership-attribute-service/app/monitoring/SentryLogging.scala:26]: this appeared to have changed in the later version of sentry lib, I just bumped it about 6 major versions (from 1.x.x. to 7.x.x).  I'm not sure we really use this functionality but I played around until it looked right in IntelliJ

File: project/Dependencies.scala [hunk 2]
```diff
@@ -66,12 +67,11 @@ object Dependencies {
     specs2.exclude("org.specs2", "specs2-mock_2.13"),
     kinesis,
     logstash,
+    logback,
     anorm,
-    "com.amazonaws" % "aws-java-sdk-autoscaling" % awsClientVersion,
```
Comment (johnduffell) [inline project/Dependencies.scala:70]: was unused, I think this was for when we used to set the request limit for attributes requests to zuora based on a fraction of the zuora request limit

File: project/Dependencies.scala [hunk 3]
```diff
@@ -66,12 +67,11 @@ object Dependencies {
...
     netty,
     nettyHttp,
     "com.google.guava" % "guava" % "32.1.3-jre", // until https://github.com/playframework/playframework/pull/10874
-    akkaHttpCore,
```
Comment (johnduffell) [inline project/Dependencies.scala:74]: was unused and we are moving to pekko now, this repo is on play 3 already

File: membership-attribute-service/app/monitoring/SentryLogging.scala (hunk 1 above)
Comment (jamesgorrie) [reply membership-attribute-service/app/monitoring/SentryLogging.scala:26]: We do sent server side errors to sentry, but I am not sure we use it. I am not sure what we get that we don't get from `CloudWatch`- maybe it's worth adding to the noise reduction effort going on?

**Review by jamesgorrie — APPROVED** [review]
Fab - nice work.

File: membership-attribute-service/app/monitoring/SentryLogging.scala (hunk 1 above)
Comment (johnduffell) [reply membership-attribute-service/app/monitoring/SentryLogging.scala:26]: I think sentry should remember all the exceptions only, group them together intelligently, and notify us somehow when something new goes wrong.  It sounds a lot better than cloudwatch's blunt metrics when used correctly.  But in reality I haven't used it, so it hasn't any value for me!

**Pull request merged**